### PR TITLE
Add hover text to icon with aasm_state

### LINKED ISSUE #343: Add hover text to icons on /box_requests/index.html.erb
Please remove the following code from the index, bc we don't want that cluttering up the BoxRequests index anymore.
```
<%= box.box_items.count %> items
<%= box.aasm_state %>
```

Instead, we'd like that whenever there's an icon showing, to add hover text displaying the aasm_state.

### PR DESCRIPTION
**Pull request by luchiago**
Resolves #343  

### Description
Is just a change to add a hover text to package icon with the aasm_state of the box.

### Type of change

- Bug fix (non-breaking change which fixes an issue)

### How Has This Been Tested?
Tested by view

### Screenshots
Before:
image

After:
image

### CODE REVIEW

File: app/views/box_requests/index.html.erb [hunk 1]
```diff
@@ -94,8 +94,7 @@
         </td>
         <td>
           <% if box %>
-            <%= box.box_items.count %> items
-            <%= box.aasm_state %>
+          <img src="https://voices-of-consent-static-images.s3.amazonaws.com/Packing+Stage.svg" title=<%= box.aasm_state %> >
```
Comment (wenderjean) [inline app/views/box_requests/index.html.erb]: What do you think to use `image_tag` helper?

https://apidock.com/rails/ActionView/Helpers/AssetTagHelper/image_tag

Comment (luchiago) [reply app/views/box_requests/index.html.erb]: That's sounds good, I will change.

Comment (luchiago) [reply app/views/box_requests/index.html.erb]: What do you think about my changes?

**Review by maebeale — APPROVED** [review]
thank you, @luchiago !!!

**Pull request merged**